build.go

```diff
@@ -320,6 +342,17 @@ func file(name string, open fileOpener) tarEntry {
 		if _, err := io.Copy(tw, f); err != nil {
 			return err
 		}
+		for _, link := range links {
+			if err := tw.WriteHeader(&tar.Header{
+				Typeflag: tar.TypeLink, // TypeSymLink?
```

Comment:
TypeSymlink is for sym links. These are hard links. What are `links ...string` supposed to be?

Reply:
renamed `links` to `hardLinks`. also verified that hard links work for our usecase on DSM7